resources/php_cs_fixer.library.dist

```diff
@@ -0,0 +1,14 @@
+<?php
```

Comment:
This configuration seems adapted for `drupal` and not for `library`

Reply:
Fixed.

Reply:
Indeed, it's fixed now.